src/util/cpupinning.cpp

```diff
@@ -0,0 +1,80 @@
+#ifdef __LINUX__
+#include <errno.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <QFile>
+#endif
+
+#include <QThread>
+#include <QtDebug>
+
+#include "util/cpupinning.h"
+
+namespace mixxx {
+
+#ifdef __LINUX__
+bool CpuPinning::moveThreadToCpuset(const QString& cgroup) {
+    qDebug() << "Move task" << gettid() << "to cpuset" << cgroup;
+    QString filename = QLatin1String("/sys/fs/cgroup/cpuset/") + cgroup + QLatin1String("/tasks");
```

Comment:
Use QStringLiteral when creating strings, this avoids utf8 to utf16 conversion at runtime.

Reply:
Yes, QLatin1String should only be used when you don't actually construct a qstring, e.g. when using comparisons like `str.startsWith(QLatin1String("foo"))`.